src/rofi_rbw/rofi_rbw.py

```diff
@@ -171,15 +185,21 @@ def get_data(self, name: str, folder: str) -> Data:
             encoding='utf-8'
         ).stdout.split('\n')
 
-        password = result[0].strip()
-        username = self.extract_username(result)
+        password = result[0]
+        username = self.extract_data("Username", result)
+        totp = self.extract_data("TOTP Secret", result)
+
+        if "pyotp" in sys.modules and totp != "":
```

Comment:
Can we do the whole import here? Like
```
try:
  import pyotp
  totp = pyotp.parse_uri(totp).now()
except:
  totp = ""
```
?

Reply:
That would work too. One argument that I can come up with to not want to import a package mid-function and just at the start of the file, could be some python structuring standards. However I am not that experienced with large python projects, thus I would say your word is law.

Reply:
I'm not sure either... it's certainly unexpected to import something in the middle of a function. On the other, the check `if "pyotp" in sys.modules` is also not exactly clear, so I'm leaning towards the import here.

Reply:
I currently moved it into the function. Maybe it would be better to do it at the start of the file and set some flag (e.g. `PYOTP_IMPORTED`) to `True`?